apps/firehose_to_es_processor/test/test_firehose_record_transmitter.py

```diff
@@ -0,0 +1,34 @@
+import unittest
+import json
+from io import BytesIO
+import gzip
+import base64
+from lib.firehose_record_transmitter import FirehoseRecordTransmitter
+from lib.firehose_record_processor import FirehoseRecordProcessor
+
+
+class TestFirehoseRecordTransmitter(unittest.TestCase):
+    region = "us-east-1"
+    stream_name = "Kinesis-Firehose-ELK-staging"
```

Comment:
If we do merge the prod branch first, we're going to have to change this

Reply:
updated